app/controllers/render-tree.js

```diff
@@ -2,16 +2,44 @@ import Ember from "ember";
 import escapeRegExp from "ember-inspector/utils/escape-reg-exp";
 import debounceComputed from "ember-inspector/computed/debounce";
 
-const { computed, isEmpty, Controller, inject: { controller } } = Ember;
+const { computed, isEmpty, Controller, inject: { controller, service } } = Ember;
 const { and, equal, filter } = computed;
 const get = Ember.get;
+let LOCAL_STORAGE_SUPPORTED;
+try {
+  LOCAL_STORAGE_SUPPORTED = !!window.localStorage;
+} catch (e) {
+  // Security setting in chrome that disables storage for third party
+  // throws an error when `localStorage` is accessed.
+  LOCAL_STORAGE_SUPPORTED = false;
+}
 
 export default Controller.extend({
   application: controller(),
   initialEmpty: false,
   modelEmpty: equal('model.length', 0),
   showEmpty: and('initialEmpty', 'modelEmpty'),
-
+  storage: service(`storage/${LOCAL_STORAGE_SUPPORTED ? 'local' : 'memory'}`),
+  // isClosed: computed(function() {
+  //   let ls = this.get('storage');
+  //   return ls.getItem('isClosed') || false;
+  // }),
+  isClosed: computed({
+    get() {
+      let ls = this.get('storage');
+      return ls.getItem('isClosed') || false;
```

Comment:
Just a tip: you can coerce `null`/`undefined` values to `false` using `!!`.
`return !!this.get('storage').getItem('isClosed');`